src/tomahawk/TomahawkWindow.cpp

```diff
@@ -769,7 +781,7 @@ TomahawkWindow::closeEvent( QCloseEvent* e )
     }
 #endif
 
-    QMainWindow::closeEvent( e );
+    QMainWindow::closeEvent(e);    
```

Comment:
The old version is the style we use throughout Tomahawk, so please change this back.